security/fma-stage-1.md

```diff
@@ -0,0 +1,127 @@
+# [Stage 1 changes]: Failure Modes and Recovery Path Analysis
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Failure Modes and Recovery Paths](#failure-modes-and-recovery-paths)
+  - [[FM1: System Cannot Be Paused]](#fm1-system-cannot-be-paused)
+  - [[FM2: System Cannot Be Unpaused]](#fm2-system-cannot-be-unpaused)
+  - [[FM3: Upgrade Process Bug]](#fm3-upgrade-process-bug)
+  - [[FM4: Invalid identifier on pause]](#fm4-invalid-identifier-on-pause)
+- [Audit Requirements](#audit-requirements)
+- [Action Items](#action-items)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+|                    |                                                    |
+| ------------------ | -------------------------------------------------- |
+| Author             | JosepBove                                          |
+| Created at         | 2025-04-06                                         |
+| Initial Reviewers  |                                                    |
+| Need Approval From |                                                    |
+| Status             |  Implementing actions                              |
+
+## Introduction
```

Comment:
Please include action items alongside each FMA instead of at the bottom, it's easier to understand what the action items are actually solving for if they are included as part of the failure mode item instead of separately at the bottom